.secretdata/config.yaml

```diff
@@ -0,0 +1,10 @@
+serviceProviders:
```

Comment:
is that test data accidentally committed?

Reply:
sure, thanks a lot!